apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -151,6 +151,9 @@ export const DashboardPage = ({
       ? "complete"
       : "incomplete";
 
+  // NOTE: Update once employee profile is complete
```

Comment:
```suggestion
  // NOTE: Update in issue #12744
```

Reply:
Referencing the issue is best https://github.com/GCTC-NTGC/gc-digital-talent/issues/12744
I updated that issue's A/C too